test/unit/atomic/src/AtomicTest.cpp

```diff
@@ -759,6 +823,24 @@ class AtomicTestKernel<
 };
 #endif
 
+#ifdef ALPAKA_ACC_ANY_BT_OACC_ENABLED
+template<typename TDim, typename TIdx, typename T>
+    class AtomicTestKernel < alpaka::AccOacc<TDim, TIdx>,
+    T,
+    std::enable_if_t<sizeof(T) <= 2> // disable 8-bit and 16-bit tests
```

Comment:
```suggestion
    std::enable_if_t<sizeof(T) <= 2>> // disable 8-bit and 16-bit tests
```
You miss an angle bracket here.

Reply:
Right, commit fixed.